### PR TITLE
Implement Reviews

### PR DESCRIPTION
**Pull request by cmfcmf**
This uses the design proposed by @etchalon and the review system discussed in #16.

Users can create one review per adventure. A review always requires a "rating" (thumbs up/down) and can optionally contain a comment.
Everyone is allowed to edit and delete their own reviews.
Curators can also delete other people's reviews.

Here's some gifs/screenshots of how it looks like:

Unreviewed adventure:
grafik

Creating a review:
44c2b2de-7119-4bc3-ac3e-8e54d3b7336d

If you already created a review:
grafik

### CODE REVIEW

File: app/Resources/webpack/js/reviews.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    $cancelReviewBtn.click(function () {
+        $addReviewBtn.removeClass('d-none');
+        $confirmReviewBtn.addClass('d-none');
+        $cancelReviewBtn.addClass('d-none');
+
+        $reviewContainer.removeClass('d-none');
+        $createReviewContainer.addClass('d-none');
+    });
+
```
Comment (burnhamrobertp) [inline app/Resources/webpack/js/reviews.js]: All of these `removeClass` / `addClass` for toggling display state can be handled a bit differently. There's always jQuery's `.toggle()` but there are other, more elegant approaches as well. For future reference.

File: app/Resources/webpack/sass/_adventure.scss [hunk 2]
```diff
@@ -4,6 +4,13 @@ $thumbnail-offset: 1.5rem;
 $thumbnail-width: 85px;
 $thumbnail-height: 100px;
 
+.subheading {
+  font: 600 1.5rem 'Arima Madurai';
+  line-height: 1.5rem;
+  color: #333333;
+  text-transform: uppercase;
+}
+
```
Comment (burnhamrobertp) [inline app/Resources/webpack/sass/_adventure.scss]: Not sure if this is a totally new font or if it was already being included or used elsewhere. If it is totally new, make sure that the resource import that loads in the fonts we're using also includes `Arima Madurai`. Also, there is a cost of loading, represented by time to download and render, that isn't free.

If its not a new font, ignore all this.

Comment (cmfcmf) [reply app/Resources/webpack/sass/_adventure.scss]: It's not a new font, it has been used elsewhere. I'll improve this by using a variable for the font though.

Comment (cmfcmf) [reply app/Resources/webpack/sass/_adventure.scss]: I've moved all occurrences of this font into a SASS placeholder class in https://github.com/AdventureLookup/AdventureLookup/pull/261/commits/e357a95ed29647f8135ce73cecc4172910e9b06d.

**Pull request merged**